script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -776,7 +777,7 @@ With vacant eyes, she slowly turns to look at me.
 [sha:b251bb168c03f7e527593ce6a36354ccf2d7ba2b]{
 -- Page 325, Offset 17380.
 -- ヘンに汗かいてるし、どう見ても重傷だぞ……！？」
-You're sweating an unreasonable amount,
+You're sweating profusely,
```

Comment:
You're dripping with sweat,
(Shiki is concerned so I can't imagine he'd be picking the eloquent words here)